### ISSUE TITLE
What happened?

### ISSUE THREAD

**Issue by agerson** [notofyi/org#532]
I love this app and it has disappeared from the app store. What happened? I would be willing to continue the development of it if you wanted to open source it.

**Comment by danielsuo** [comment]
Appreciate the kind words! Responding in email

**Comment by mlisseman** [comment]
Yes! I found this app insanely useful! It's still working on my 6s (bar the long string of random characters in the Subject line), but I can't get it on my new phone. Devastated!

**Comment by danielsuo** [comment]
Sadly, I don't have the time to give it proper attention. A new version of iOS caused the subject strings to be read improperly (hence the random characters). New app store procedures. Etc.

@agerson recommended an app called "Email Me" that's been working out nicely. It has many of the features that I wanted to add to Noto.

Only thing is...I wish they'd move the send button from the top-right corner to just above the keyboard. Sounds silly, but adds a decent amount of friction.

**Comment by mlisseman** [comment]
That’s cool. I’ll check out that other app (although the dev’s website won’t load in Safari - it’s just blank!)

Thanks
Mark

**Comment by rcach001** [comment]
Bring it back, with voice too! 😄 

What can we do? Maybe crowdsource the development and maintenance?

**Comment by danielsuo** [comment]
If I look at history, Noto 1 came out in 2014 before it died. Noto 2 was a full rebuild and came out in early 2018. It died a similar death recently. Perhaps we're due for a Noto 3?

Don't have any plans for it, but if folks want to work on a version 3, would be happy to start a repository and chat about it. Strong preference to build from scratch because the app is really simple and was just me learning React Native so the Noto 2 codebase is a dumpster fire. Also would want to think about a more sustainable revenue structure (as much as I personally hate subscribing to things...).

**Comment by agerson** [comment]
A good alternative: https://apps.apple.com/us/app/email-me-notes-in-your-inbox/id1090744587

**Comment by danielsuo** [comment]
Indeed. This is what I currently use (see previous comments)
